proposals/compatibility-mode.md

```diff
@@ -103,12 +119,16 @@ Calls to `createTexture()` or `createBindGroupLayout()` with this combination ca
 
 **Justification**: GLSL ES 3.1 (section 4.4.7, "Format Layout Qualifiers") does not permit any two-component (RG) texture formats in a format layout qualifier.
 
+Feature: `storage-texture-rg`.
```

Comment:
Maybe `storage-texture-rg32` since these are all incidentally rg32.

Reply:
Good point. I've made it `rg32-storage-texture` to match other features, where the format comes first.